sympy/matrices/common.py

```diff
@@ -1529,7 +1529,6 @@ def is_anti_symmetric(self, simplify=True):
         simplification the matrix does not appear anti-symmetric:
 
         >>> m.is_anti_symmetric(simplify=False)
```

Comment:
```suggestion
        >>> print(m.is_anti_symmetric(simplify=False))
        None
```

This could try to use `print` instead of being invisible